include/swift/Frontend/Frontend.h

```diff
@@ -433,6 +444,65 @@ class CompilerInstance {
   /// Frees up the ASTContext and SILModule objects that this instance is
   /// holding on.
   void freeContextAndSIL();
+
+private:
+  bool loadStdlibAndMaybeSwiftOnoneSupport();
+  ModuleDecl *importUnderlyingModule(ClangImporter *clangImporter);
+  ModuleDecl *importBridgingHeader(ClangImporter *clangImporter);
+
+  void fillInModulesToImportFromImplicitImportModuleNames(
+      SmallVectorImpl<ModuleDecl *> &importModules);
+  
+public: // for static functions in Frontend.cpp
+  
+  struct ImplicitImports {
+    SourceFile::ImplicitModuleImportKind kind;
+    ModuleDecl *objCModuleUnderlyingMixedFramework;
+    ModuleDecl *headerModule;
+    SmallVector<ModuleDecl *, 4> modules;
+    
+    ImplicitImports(CompilerInstance &compiler);
+  };
+  
+private:
+  void createREPLFileWithImports(ImplicitImports &implicitImports);
+  DelayedParsingCallbacks *computeDelayedParsingCallback();
+
+  void createMainFileWithImportsAndAddToModule(ImplicitImports &implicitImports);
+ 
+  void parseALibraryFile(
```

Comment:
I'm going to say the "a" should go. :-) Singular "file" is sufficient.

Reply:
Done. Love the attention to detail.